pkg/volume/volume_info_common.go

```diff
@@ -0,0 +1,102 @@
+/*
+Copyright 2018 the Velero contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package volume
+
+type VolumeInfoVersion struct {
+	Version string `json:"version"`
+}
+
+const (
+	VolumeInfoVersionV1 string = "1"
+)
+
+type VolumeInfos struct {
+	VolumeInfosV1 []VolumeInfoV1
+	Version       string
+}
+
+// CSISnapshotInfo is used for displaying the CSI snapshot status
+type CSISnapshotInfo struct {
+	// It's the storage provider's snapshot ID for CSI.
+	SnapshotHandle string `json:"snapshotHandle"`
+
+	// The snapshot corresponding volume size. Some of the volume backup methods cannot retrieve the data by current design, for example, the Velero native snapshot.
+	Size int64 `json:"size"`
+
+	// The name of the CSI driver.
+	Driver string `json:"driver"`
+
+	// The name of the VolumeSnapshotContent.
+	VSCName string `json:"vscName"`
+}
+
+// SnapshotDataMovementInfo is used for displaying the snapshot data mover status.
+type SnapshotDataMovementInfo struct {
```

Comment:
A repo snapshot ID is also required for data mover

Reply:
Added.